src/components/Editor/SearchBar.js

```diff
@@ -308,10 +316,19 @@ class SearchBar extends Component {
 
     if (modifiers) {
       const findFnc = rev ? findPrev : findNext;
-      const newIndex = findFnc(ctx, query, true, modifiers.toJS());
+      const findPos = findFnc(ctx, query, true, modifiers.toJS());
+      const { ch: newIndex } = findPos;
+      const matchedLocationIndex = matchedLocations
```

Comment:
It might be nice to move this function to a helper: `getSearchIndex(currentPos, matchedLocations)`